app/views/virtual_workshop_invitation_mailer/attending.html.haml

```diff
@@ -15,14 +15,17 @@
                 %h3 Hi #{@member.name},
                 %p.lead
                   - if @waiting_list
-                    A spot became available and your attendance has now been confirmed.
+                    A spot became available and your attendance has now been confirmed!
                   - else
-                    Your spot has been confirmed.
+                    Your spot has been confirmed!
                     - if @invitation.role.eql?('Student')
-                      %p If you haven't already, please remember to let us know what you will be working on through the invitation. Projects that involve specific frameworks, libraries, or languages that are not part of the codebar tutorials (anything other than Ruby or Javascript) are welcome, but we cannot guarantee that a coach will be available to help you. We recommend having a backup option to work on, such as codebar tutorials or katas (codewars.com or exercism.io). See you at our workshop!
+                      %p If you haven't already, please let us know what you will be working on through the invitation. Projects that involve specific frameworks, libraries, or languages that are not part of the codebar tutorials (anything other than Ruby or Javascript) are welcome, but we cannot guarantee that a coach will be available to help you. We recommend having a backup option to work on, such as codebar tutorials or katas (codewars.com or exercism.io).
```

Comment:
@KimberleyCook should we make codewars.com and exercism.com actual links here?

Reply:
I don't see why we'd want to send people to their website, but I'm happy to amend?

Reply:
@KimberleyCook I just think it's weird that they are styled as links (e.g. include the .com part) but are not actually links.